types/graphql-schema-linter/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for graphql-schema-linter 0.2
+// Project: https://github.com/cjoudrey/graphql-schema-linter
+// Definitions by: Gago <https://github.com/gagoar>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace GraphqlSchemaLinter {
```

Comment:
I wrote it there, because I was reusing it in other types. I'm correcting this, and avoiding the export, and just have the interface to use inside the `Configuration.d.ts`. 
leme know if there's a better or more cannonical way to do it.